packages/app-project/pages/_app.js

```diff
@@ -19,8 +19,21 @@ const GlobalStyle = createGlobalStyle`
     margin: 0;
   }
 `
-
-enableStaticRendering(typeof window === 'undefined')
+const isBrowser = typeof window !== 'undefined'
+const localStorage = isBrowser ? window.localStorage : null
+enableStaticRendering(isBrowser)
```

Comment:
I think this is why the classifier is broken. The store should be static on the server, dynamic in the browser. 

```suggestion
enableStaticRendering(!isBrowser)
```